CONTRIBUTING.md

```diff
@@ -2,8 +2,30 @@
 
 The team behind the [OSS Review Toolkit](https://github.com/heremaps/oss-review-toolkit) gratefully accepts contributions via
 [pull requests](https://help.github.com/articles/about-pull-requests/) filed against the
-[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls). As part of filing a pull request we ask you to
-sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
+[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls).
+
+# Signing each Commit
+
+As part of filing a pull request we ask you to
+sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) in each commit.
+Any Pull Request with commits that are not signed off will be reject by the
+[DCO check](https://probot.github.io/apps/dco/).
+
+A DCO is lightweight way for a contributor to confirm that they wrote or otherwise have the right
+to submit code or documentation to a project.
+Simply add `Signed-off-by` as shown in the example below to indicate that you agree with the DCO.
```

Comment:
This sentence should also already start at line 14 as in the output no paragraph / line break is created unless also a blank line is added. I.e. making this sentence start at line 14 makes the visual appearance of this text and rendered output more similar, which is a good thing.

Reply:
Ping...

Reply:
Done